app/assets/stylesheets/application/_settings/_colors.scss

```diff
@@ -45,7 +45,7 @@ $brand-danger-l:        #ffa69b !default;
 // COB-sg colors
 $cob-sg-freedom-trail-red: rgb(251, 77, 66) !default; // #FB4D42
 $cob-sg-charles-blue: rgb(9, 31, 47) !default; // #091F2F
-$cob-sg-optimistic-blue: rgb(40, 139, 238) !default; // #288BE4
+$cob-sg-optimistic-blue: #60AAEB !default;
```

Comment:
note to self: need darken this again because it's used as a link on white background, even thought it messes up the enrollment calendar